sql/testdata/aggregate

```diff
@@ -165,33 +211,16 @@ SELECT MAX(k), MIN(v) FROM kv HAVING MAX(v) > 2;
 query error argument of HAVING must be type bool, not type int
 SELECT MAX(k), MIN(v) FROM kv HAVING k;
 
-query III rowsort
-SELECT k, 3, MAX(v) FROM kv GROUP BY k HAVING k > 5;
-----
-6 3 2
-7 3 2
-8 3 4
-
-# The presence of a HAVING triggers aggregation, reducing the un-grouped query to single bucket.
-query I rowsort
-SELECT 3 FROM kv WHERE k > 5 HAVING TRUE;
-----
-3
-
-query I rowsort
-SELECT 3 FROM kv WHERE k > 5;
-----
-3
-3
-3
+# Columns listed in the HAVING must conform to same validation as SELECT (grouped or aggregated)
+query error column "k" must appear in the GROUP BY clause or be used in an aggregate function
+SELECT 3 FROM kv GROUP BY v HAVING k > 5;
 
+# pg has a special case for grouping on primary key, which would allow this, but we do not.
+# See http://www.postgresql.org/docs/current/static/sql-select.html#SQL-GROUPBY
 query error column "v" must appear in the GROUP BY clause or be used in an aggregate function
 SELECT 3 FROM kv GROUP BY k HAVING v > 2
 
 query error column "k" must appear in the GROUP BY clause or be used in an aggregate function
-SELECT 3 FROM kv GROUP BY v HAVING k > 5;
```

Comment:
keep this one. if someone decides to implement the special case above, they're gonna need this test.